tsconfig-demos.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "./tsconfig-base",
-  "include": ["src/demos", "src/components.d.ts"]
+  "include": ["src/demos", "src/components.d.ts"],
+  "exclude": ["node_modules"]
```

Comment:
Nitpick: `node_modules` is already excluded by default: https://www.typescriptlang.org/tsconfig#exclude